libsvm-kitware/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd TPL/libsvm-3.1-custom
+
+make lib
+mkdir $PREFIX/lib
+mv libsvm* $PREFIX/lib/
```

Comment:
You are right, I do need to make a new release soon. My current context is coming from the master branch. I will try to get to that in the next day or so to make sure things are in order.

As for When I do a ``find . -name "svm.py"`` from SMQTK's source (on master) it yields ``./TPL/libsvm-3.1-custom/python/svm.py``, which is in the libSVM source tree (same place as the original libSVM repo that the custom source tree is based on (https://github.com/cjlin1/libsvm)). The same should be the case for the v0.1 tag (just checked it out and checked). On that tag there is also a reference in the ``OLD_ROOT`` sub-directory, but we can ignore everything in that (removed on master and in the coming release).